cmd/mysql/delete.go

```diff
@@ -54,7 +54,7 @@ var deleteTargetCmd = &cobra.Command{
 
 type DeleteHandler struct {
 	*internal.DeleteHandler
-	permanentObjects map[string]bool
+	permanentObjects []string
```

Comment:
Why are you changing the map here for slice?
Map was used for search, didn't it?

Reply:
The map wasn't used for search here. It's only provided to `internal.DeleteHandler.HandleDeleteEverything`. And that function uses this only to log backup names and nothing else. The reason why it now accepts slice instead of map is that for DBs that support multiple storages, backup name is not a unique identifier anymore, because backups with the same name can be stored in different storages. I could actually make the key type of this map a special struct like `{backupName, storageName}`, but I just noticed that map isn't really necessary here and slice is also OK, so I made it slice.